### DISCUSSION TITLE
Certain top menu app icons seem to contain no path to app url

### DISCUSSION THREAD

**Discussion by emelbardis** [nextcloud/all-in-one#5153] [Questions] [answered]
Noticed this a couple days ago. My Tasks and Memories menu icons don't take me to my Tasks or Memories apps but instead just refresh the page I'm on. Changing the url manually in browser opens up those apps just fine while clicking on the menu app icon does not. Is there a way to repair or rebuild these broken or null paths?

Any info I can share to help troubleshoot this?

**Comment by ebildebil** [comment, marked as answer]
Hi,

I was trouble shooting this just today. In my case I did as follows:
Administarive settings -> Full Text Search-> General

There uncheck the "Navigation Icon"

That fixed it, however, at the expense of Global full text search.

**Reply by emelbardis** [reply]
Thank you. I don't recall ticking that box but unchecking it brought back my paths for those apps
